Content.Shared/Chemistry/Reagent/ReagentPrototype.cs

```diff
@@ -16,14 +19,14 @@ namespace Content.Shared.Chemistry.Reagent
     [DataDefinition]
     public class ReagentPrototype : IPrototype
     {
+        [DataField("metabolisms", serverOnly: true)] //customTypeSerializer: PrototypeIdDictionarySerializer<List<ReagentEffectsEntry>, PrototypeIdListSerializer<MetabolismGroupPrototype>>)]
```

Comment:
Does the custom type serializer not work?

Reply:
nope

Reply:
nvm